docs/CONFIG.md

```diff
@@ -220,7 +220,7 @@ file referenced above:
 
 ```ini
 [session]
-session_type = "lpass"
+session_type = lpass
```

Comment:
This isn't related to the theme of this PR, but I found it along the way - our config parser doesn't use double quotes for parsing in its .ini format.

Reply:
Huh. Good find.